Code/MinimalLib/cffi_test.c

```diff
@@ -1010,6 +1010,97 @@ void test_standardize() {
   printf("--------------------------\n");
 }
 
+void test_get_mol_frags() {
```

Comment:
looking at all of this C code makes my eyes burn.

Thanks for suffering through and creating these tests!